modules/splunk-configuring-acs.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-splunk.adoc
+:_module-type: PROCEDURE
+[id="splunk-configuring-acs_{context}"]
+= Configuring {product-title}
```

Comment:
Same as other integration PRs, a more descriptive title I think would be helpful.